README.md

```diff
@@ -110,6 +110,8 @@ Options:
 - `maxHeaderSize: Number`, the maximum length of request headers in bytes.
   Default: `16384` (16KiB).
 
+- `strictContentLength: Boolean`, whether to treat request content length mismatches as errors. If true, an error is thrown when the request content-length header doesn't match the length of the request body. Default: `true`.
```

Comment:
Can you add this to the `docs/Client.md` file as well please?

Reply:
done